testsuite/features/support/commonlib.rb

```diff
@@ -32,19 +32,20 @@ def compute_channels_to_leave_running
   # keep the repos needed for the auto-installation tests
   do_not_kill =
     if product == 'Uyuni'
-      CHANNEL_TO_SYNCH_BY_OS_VERSION['15.4']
+      CHANNEL_TO_SYNCH_BY_OS_PRODUCT_VERSION['sles15.4']
```

Comment:
```suggestion
      CHANNEL_TO_SYNCH_BY_OS_PRODUCT_VERSION['leap15.4-x86_64']
```
This is wrong. It must be Leap instead of SLES.

Reply:
This is not wrong, this is just updating the name of what we has been using until today.
I will not change it to leap in this PR, because this must be treat in a different PR.
In theory, you are working a PR for that change.

Reply:
It has to go with this PR, since you changed the behavior. I will mark the code in a comment below.

Reply:
Ah ok, I see! Ok, fixing.

Reply:
Thank yoU!